index.html

```diff
@@ -0,0 +1,220 @@
+            </g>
+        </symbol>
+    </svg>
+
+    <svg display="none">
+        <symbol id="management" width="78.969" height="91" fill="#3c3c3c" viewBox="0 0 78.969 91">
+            <g>
+                <path d="M939.756,3177.11a1.579,1.579,0,0,1-1.109-.45l-3.1-3.08a1.548,1.548,0,0,1,0-2.2,1.574,1.574,0,0,1,2.219,0l1.989,1.97,5.086-5.05a1.574,1.574,0,0,1,2.219,0,1.546,1.546,0,0,1,0,2.2l-6.2,6.16A1.589,1.589,0,0,1,939.756,3177.11Zm32.555-3.07H953.637a1.56,1.56,0,1,1,0-3.12h18.674A1.56,1.56,0,1,1,972.311,3174.04Zm-32.555,29.75a1.579,1.579,0,0,1-1.109-.45l-3.1-3.08a1.548,1.548,0,0,1,0-2.2,1.574,1.574,0,0,1,2.219,0l1.989,1.97,5.086-5.05a1.574,1.574,0,0,1,2.219,0,1.546,1.546,0,0,1,0,2.2l-6.2,6.16A1.589,1.589,0,0,1,939.756,3203.79Zm32.555-3.07H953.637a1.56,1.56,0,1,1,0-3.12h18.674A1.56,1.56,0,1,1,972.311,3200.72Zm-32.555,16.42a1.589,1.589,0,0,1-1.109-.46l-3.1-3.08a1.548,1.548,0,0,1,0-2.2,1.574,1.574,0,0,1,2.219,0l1.989,1.97,5.086-5.05a1.574,1.574,0,0,1,2.219,0,1.557,1.557,0,0,1,0,2.21l-6.2,6.15A1.6,1.6,0,0,1,939.756,3217.14Zm32.555-3.08H953.637a1.56,1.56,0,1,1,0-3.12h18.674A1.56,1.56,0,1,1,972.311,3214.06Zm-32.555-23.61a1.579,1.579,0,0,1-1.109-.45l-3.1-3.08a1.56,1.56,0,0,1,0-2.21,1.593,1.593,0,0,1,2.219,0l1.989,1.98,5.086-5.05a1.574,1.574,0,0,1,2.219,0,1.545,1.545,0,0,1,0,2.2l-6.2,6.16A1.589,1.589,0,0,1,939.756,3190.45Zm32.555-3.07H953.637a1.56,1.56,0,1,1,0-3.12h18.674A1.56,1.56,0,1,1,972.311,3187.38Zm22.577-36.98H976.422a6.132,6.132,0,0,0-6.141-5.68H964.3a5.812,5.812,0,0,0-11.623,0h-5.994a6.128,6.128,0,0,0-6.131,5.68H920.081a2.088,2.088,0,0,0-2.092,2.08v75.44a2.082,2.082,0,0,0,2.092,2.08h74.807a2.083,2.083,0,0,0,2.093-2.08v-75.44A2.089,2.089,0,0,0,994.888,3150.4Zm-50.177.44a1.965,1.965,0,0,1,1.968-1.96h6.539a3.63,3.63,0,0,0,3.641-3.61v-0.48a1.627,1.627,0,0,1,3.254,0v0.48a3.627,3.627,0,0,0,3.63,3.61h6.539a1.966,1.966,0,0,1,1.977,1.96v4.26H944.713v-4.26h0Zm48.085,75H922.173v-71.28h18.353v2.62a2.082,2.082,0,0,0,2.092,2.08h31.731a2.082,2.082,0,0,0,2.092-2.08v-2.62h16.353v71.28h0Zm-34.311-76.7a2,2,0,1,1-2.01,2A2,2,0,0,1,958.485,3149.14Z" transform="translate(-918 -3139)"/>
+            </g>
+        </symbol>
+    </svg>
+
+    <svg display="none">
```

Comment:
And for all SVG's in HTML - try to create and use SVG sprite with big graphic code